apple/Handlers/RNLongPressHandler.m

```diff
@@ -94,6 +107,83 @@ - (void)touchesCancelled:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)ev
   [self reset];
 }
 
+#else
+- (void)mouseDown:(NSEvent *)event
+{
+  self.state = NSGestureRecognizerStateBegan;
+  startTime = CACurrentMediaTime();
```

Comment:
`previousTime` isn't set anywhere so the duration probably isn't right

Reply:
It seemed ok but I'll check that

Reply:
Actually it is set inside `handleGesture` method 😅